File: Data Structures & Algorithms/two-integer-sum/submission-1.java
class Solution {
    public int[] twoSum(int[] nums, int target) {
       for(int o = 0; o < nums.length; o++){
        for(int i = o+1; i < nums.length; i++){
            if(nums[i] + nums[o]==target) return new int[]{o,i};
        }
       }
       return new int[]{0,0};
    }
}

File: Data Structures & Algorithms/top-k-elements-in-list/submission-1.py
class Solution:
    def topKFrequent(self, nums: List[int], k: int) -> List[int]:
        freqs = {}
        for x in nums:
            if x in freqs:
                freqs[x] += 1
            else:
                freqs[x] = 1
        bucketSort = [[] for _ in range(len(nums))]
        for x in freqs:
            bucketSort[freqs[x]-1].append(x)
        ret, filledOut = [0]*k, 0
        for x in range (len(bucketSort)-1, -1, -1):
            for val in bucketSort[x]:
                ret[filledOut] = val
                filledOut += 1
                if filledOut>=k:
                    return ret
        return ret
